src/azul/service/avro_pfb.py

```diff
@@ -602,3 +642,129 @@ def _entity_schema_recursive(field_types: FieldTypes,
             pass
         else:
             assert False, field_type
+
+
+def _update_replica_schema(*,
+                           schema: MutableJSON,
+                           path: Sequence[str],
```

Comment:
```suggestion
                           path: tuple[str, ...],
```

So that no one accidentally passes a list.